Authentication/Extensions.swift

```diff
@@ -16,11 +16,11 @@ internal extension String {
          not have the key in the framework one.
      */
     internal var frameworkLocalized: String {
-        let localized: (from: NSBundle) -> String = {
+        let localized: (_ from: Bundle) -> String = {
```

Comment:
But with a `CVArgs...` parameter, you can give no arguments. At least you could in Swift 2. strange